client/app/components/routes/access-requests/form/UserOperatorReview.tsx

```diff
@@ -90,6 +119,9 @@ export default function UserOperatorReview({
         isOperatorNew ? approveOperatorRequest : approvePrimeAdminRequst
       }
       onReject={isOperatorNew ? rejectOperatorRequest : rejectPrimeAdminRequest}
+      showRequestChanges={isOperatorNew}
```

Comment:
@andrea-williams hey just updating this after rebasing and wanted to make sure this is correct based on brief discussion at standup. We only want to be able to request changes for the operator, not the prime admin so I believe this is the logic we need?

If the operator exists already then we don't want to request changes either?

Reply:
Nevermind I figured this out and passed in a prop `isPrimeAdminRequest` so we would know to hide it for that.